.changeset/dirty-yaks-own.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": patch
+---
+
+change test description for new moonpay description
```

Comment:
I would remove this changeset and change the title of the PR to `test` instead of `feat` as we're only modifying a mock file and nothing needing a new version for the user